docs/operations.md

```diff
@@ -165,6 +165,27 @@ For more information on profiling see the excellent [Profiling Go Programs](http
 * can use debug logging to trace data throughout the pipeline. mt-store-cat to see what's in cassandra, mt-kafka-mdm-sniff, etc.
 * if it's old data, make sure you have a primary that can save data to cassandra, that the write queue can drain
 * check `metric-max-stale` and `chunk-max-stale` settings, make sure chunks are not being prematurely sealed (happens in some rare cases if you send data very infrequently. see `tank.add_to_closed_chunk` metric)
+* did you restart instances? if so: make sure your instances start replaying data within the allotted "overhead window". E.g. if your kafka retention is 7 hours and your largest chunks are 6 hours, then instances need to start replaying data within an hour after startup. (so make sure processing of metricpersist messages, index loading, etc doesn't take too long). Any subsequent restart (e.g. due to kafka removing a segment currently being consumed) starts the process from zero again, so watch out.   Increase kafka retention as needed.
+
+In the below example, we:
+* use a max chunk span of 6 hours
+* use 30 minutes as time needed to get all chunks saved to the store
+* assume startup until data backfill takes about 1 hour (see [startup procedure](startup.md))
+* assume 7.5 hours Kafka retention
+
+As you can see, you need at least 7.5 hours of Kafka retention to support this case.
+(note: for simplicity, we don't include time needed to process initial segment which may be dropped by Kafka)
+
+```
+
+                                                          | 05:00 | 06:00 | 07:00 | 08:00 | 09:00 | 10:00 | 11:00 | 12:00 | 13:00 |
+                                                          |-----------------------------------------------------------------------|
```

Comment:
Sorry, after approving the PR i just noticed that the formatting of this seems to be kind of broken:

https://github.com/grafana/metrictank/blob/f78c793c1d963c04145367d58f81783d9f0d5a1e/docs/operations.md#data-doesnt-show-up

Reply:
@replay can you be more specific. it looks just like the md as far as i can see

Reply:
nvm, i looked at it again and i understand it now